src/component/App.css

```diff
@@ -0,0 +1,9 @@
+body {
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+}
+
+.app {
```

Comment:
Why do we have different classes styles `.app` and `.Footer`?

Reply:
???